src/etc/ssl/opnsense.cnf

```diff
@@ -292,3 +292,14 @@ extendedKeyUsage       = clientAuth,serverAuth,1.3.6.1.5.5.8.2.2
 keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
 
 ###OPNsense:subjectAltName###
+
+[ sign_csr ]
+
+nsComment              = "OPNsense Generated Combined Client/Server Certificate"
```

Comment:
Seems wrong to me. Why combined client/server certificate?! This is controlled by `keyUsage` and `extendedKeyUsage` instead, I believe.

Reply:
Oh, sorry, I did not read well and thought all are the same. I will remove `Combined Client/Server`.